core/src/main/scala/org/apache/spark/storage/BlockManager.scala

```diff
@@ -454,14 +454,20 @@ private[spark] class BlockManager(
       case Some(info) =>
         val level = info.level
         logDebug(s"Level for block $blockId is $level")
+        val taskAttemptId = Option(TaskContext.get()).map(_.taskAttemptId())
```

Comment:
Hi, @jiangxb1987. Looks like the same way to get `taskAttemptId` as `BlockInfoManger`. What's the difference to get the `taskAttemptId` instead of `BlockInfoManager.currentTaskAttemptId` ?